src/components/accordion/Accordion.jsx

```diff
@@ -77,9 +143,21 @@ const Accordion = ({
 
   return (
     <AccordionContext.Provider
-      value={{noGapBetweenElements, opened: state.opened, dispatch}}
+      value={{
+        noGapBetweenElements,
+        opened: state.opened,
+        focusedElementId: state.focusedElementId,
+        dispatch,
+      }}
     >
-      <div className={cx(spaceClass, className)}>{children}</div>
+      <div
+        ref={wrapperRef}
+        className={cx(spaceClass, className)}
+        data-allow-multiple={allowMultiple}
```

Comment:
sorry if i missed sth but are these two: `data-allow-multiple`, `data-allow-toggle` used somehow? or there is some a11y for having them?

Reply:
used by screen readers

Reply:
👏 thanks! perfect!